### PR TITLE
[Bugfix] Fix the SSE tutorial in CI.

### PR DESCRIPTION
**Pull request by zheng-da**
## Description
It seems the SSE tutorial fails undeterministically. It's probably caused by the empty neighbor list in the neighbor sampling.

## Checklist
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [$CATEGORY] (such as [Model], [Doc], [Feature]])
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [ ] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

### CODE REVIEW

File: src/graph/sampler.cc [hunk 1]
```diff
@@ -320,8 +320,10 @@ NodeFlow ConstructNodeFlow(std::vector<dgl_id_t> neighbor_list,
       dgl_id_t dst_id = sub_vers->at(i).first;
       assert(dst_id == neigh_pos->at(i).id);
       size_t pos = neigh_pos->at(i).pos;
-      CHECK_LT(pos, neighbor_list.size());
+      CHECK_LE(pos, neighbor_list.size());
       size_t num_edges = neigh_pos->at(i).num_edges;
+      if (neighbor_list.empty())
+        CHECK(num_edges == 0);
```
Comment (jermainewang) [inline src/graph/sampler.cc]: If you really like having no brace for one line of statement, put them in one line.

File: src/graph/sampler.cc [hunk 2]
```diff
@@ -320,8 +320,10 @@ NodeFlow ConstructNodeFlow(std::vector<dgl_id_t> neighbor_list,
       dgl_id_t dst_id = sub_vers->at(i).first;
       assert(dst_id == neigh_pos->at(i).id);
```
Comment (jermainewang) [inline src/graph/sampler.cc]: assert <- CHECK

**Comment by jermainewang** [comment]
Please change the PR name to [Bugfix] xxx .

File: src/graph/sampler.cc (hunk 1 above)
Comment (zheng-da) [reply src/graph/sampler.cc]: this is simply a programming habit. i have been doing this for many years. it's not likely to change. i think this kind of variance is ok as long as it meets the google coding style.

Comment (BarclayII) [reply src/graph/sampler.cc]: My habit (largely follows [Linux](https://www.kernel.org/doc/html/v4.18/_sources/process/coding-style.rst.txt)):

```C
/* good */
if (condition)
    one-line statement;

/* good */
if (condition) {
    one-line statement;
}

/* no */
if (condition) one-line statement;

/* no */
if (condition)
{
    one-line statement;
}

/* no-no-no */
if (condition) one-line statement;
    one-line statement that is always executed;
```

**Pull request merged**